components/file_attachment_list/file_attachment_list.jsx

```diff
@@ -67,25 +67,28 @@ export default class FileAttachmentList extends React.Component {
 
     render() {
         const {fileInfos, fileCount, compactDisplay} = this.props;
-        const postFiles = [];
-        let sortedFileInfos = [];
 
-        if (fileInfos && fileInfos.length === 1 && compactDisplay === false) {
-            const fileType = getFileType(fileInfos[0].extension);
-
-            if (fileType === FileTypes.IMAGE || fileType === FileTypes.SVG) {
+        if (compactDisplay === false) {
+            if (fileInfos && fileInfos.length === 1) {
+                const fileType = getFileType(fileInfos[0].extension);
+
+                if (fileType === FileTypes.IMAGE || fileType === FileTypes.SVG) {
+                    return (
+                        <SingleImageView
+                            fileInfo={fileInfos[0]}
+                        />
+                    );
+                }
+            } else if (fileCount === 1) {
                 return (
-                    <SingleImageView
-                        fileInfo={fileInfos[0]}
-                    />
+                    <div style={{minHeight: '385px'}}/>
```

Comment:
Not a stopper at all, but I don't love this kind of "inline" styles, I would prefer a class and something in the CSS. Anyway, being a fix for 4.7, if we want to change it, we can create a ticket for it and merge this PR as is. Probably here is interesting the @asaadmahmood opinion about it.

Reply:
I agree, we should assign it a class or something for that matter.

Reply:
Thanks! I added style object for that purpose. See updates.